pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -1,6 +1,7 @@
 { fetchurl, lib, stdenv, buildPackages
 , curl, openssl, zlib, expat, perlPackages, python3, gettext, cpio
 , gnugrep, gnused, gawk, coreutils # needed at runtime by git-filter-branch etc
+, withSshSupport ? true
```

Comment:
`withSshSupport` is misleading. SSH is supported either way, this is just about using a fixed SSH binary by hard-coding the Nix store path.